packages/core/src/login/oidc/redirectHandler/IRedirectHandler.ts

```diff
@@ -33,5 +33,8 @@ export type RedirectResult = ISessionInfo & { fetch: typeof fetch };
 /**
  * @hidden
  */
-type IRedirectHandler = IHandleable<[string], RedirectResult>;
+type IRedirectHandler = IHandleable<
+  [url: string, handleRefreshToken?: (token: string) => unknown],
```

Comment:
how does the `handleRefreshToken?` method differ from `handleRefreshTokenRotation?`

Reply:
In essence, they don't, and I'd have preferred if using only one would be sufficient. However, they are used in two different contexts: since they aren't tied to the session, they need to be injected in the two entry points when they may be used, that is either when the refresh token is received from the token endpoint initially (hence when calling `handleIncomingRedirect`), or whenever a refresh token is used to get a new access token (hence when calling `login`).

However, writing this, I realize the callback could be specified as part of the `Session` constructor, and then injected behind the scenes by the `Session` object to the calls to the `ClientAuthorization` methods. I'll make a PR in that sense, it will make a cleaner API.